docker/build-tools/Dockerfile

```diff
@@ -643,7 +643,7 @@ ENV RUSTUP_HOME=/home/.rustup
 # hadolint ignore=DL4006
 RUN curl --proto '=https' -v --tlsv1.2 -sSf https://sh.rustup.rs | \
     sh -s -- -y -v --default-toolchain ${RUST_VERSION} --profile minimal \
-    --component rustfmt clippy --component llvm-tools &&\
+    --component rustfmt --component clippy --component llvm-tools &&\
```

Comment:
Is this the fix we want (to mirror the change on master in https://github.com/istio/tools/pull/3157) and if so is it worth updating release-1.24 which was fixed differently in https://github.com/istio/tools/commit/7d1c19cf1d83ca3cfdb7cc7b90dd807fe41653da for consistency?

Refs context in https://github.com/istio/tools/pull/3160#issuecomment-2738308914

Reply:
I don't think we need to touch release 1.24. this fails without so I mirrored master.